web3/account.py

```diff
@@ -60,14 +58,11 @@ class Account(object):
     def create(self, extra_entropy=''):
         extra_key_bytes = text_if_str(to_bytes, extra_entropy)
         key_bytes = keccak(os.urandom(32) + extra_key_bytes)
-        if sys.version_info.major < 3:
-            key_bytes = to_hex(key_bytes)
         return self.privateKeyToAccount(key_bytes)
 
     @staticmethod
     def decrypt(keyfile_json, password):
-        if isinstance(keyfile_json, str) or (
-                sys.version_info.major < 3 and isinstance(keyfile_json, unicode)):  # noqa: 821
+        if isinstance(keyfile_json, str):  # noqa: 821
```

Comment:
Why `noqa` here?

Reply:
@techtonik that was already in the code so I just left it there.  I'm not sure what the comment is referencing.

Reply:
Just FYI, these noqa comments are a way of telling `flake8` not to issue the specified warning. Most of the time it's best to fix the thing causing the warning, but occasionally the best thing to do is suppress it.

The list of warnings are here: http://flake8.pycqa.org/en/3.4.1/user/error-codes.html#error-violation-codes

821 is usage of an undefined name, which is usually pretty bad, but in this case it was just because `unicode` isn't defined in py3, and we know a py3 codepath to that `unicode` in this line is impossible.

Now that `unicode` is removed, we can and should remove the warning supressor.